.semaphore/semaphore.yml

```diff
@@ -65,15 +65,19 @@ blocks:
       prologue:
         commands:
           - checkout && cd ~/git/flowcrypt-ios/ && cache restore && make dependencies && cache store
-          - sem-version node 16 && mkdir .custom-npm && cd .custom-npm && npm install npm@8.1.2 && cd ..
-          - ( cd Core && cache restore core-npm && ../.custom-npm/node_modules/.bin/npm install && cache store core-npm node_modules )
+          - npm install -g npm@8.11.0
```

Comment:
Why do we use such old node version here? I tried to build `Core` locally with `node@16` - works well, without any errors

Reply:
oops, misread it 🙂
then all good

Reply:
8.11.0 is latest npm version